Aim: To estimate the true mean durability (in millions of characters) print-heads using a small sample of size 15, and calculate 99% confidence intervals using:
1. Sample standard deviation(use t-distribution)
2. Known population standard deviation = 0.2 (use z-distribution)

# ***Sample Data***

In [ ]:
data = [1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]


# **1. Calculate basic sample statistics**

In [ ]:
import numpy as np

data = [1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]

n = len(data)
mean = np.mean(data)
sample_std = np.std(data, ddof=1)

print(f"Sample Size: {n}")
print(f"Sample Mean: {mean:.4f}")
print(f"Sample Standard Deviation: {sample_std:.4f}")

Sample Size: 15
Sample Mean: 1.2387
Sample Standard Deviation: 0.1932


# **2a Construct 99% confidence interval(when std dev is unknown)**

In [ ]:
from scipy.stats import t

confidence = 0.99
alpha = 1 - confidence
df = n - 1
t_crit = t.ppf(1 - alpha/2, df)

margin_error = t_crit * (sample_std / np.sqrt(n))
ci_lower = mean - margin_error
ci_upper = mean + margin_error

print(f"99% Confidence Interval (t-distribution): ({ci_lower:.4f}, {ci_upper:.4f})")


99% Confidence Interval (t-distribution): (1.0902, 1.3871)


# **Step 2b: Construct 99% Confidence Interval (when std dev is known)**

In [ ]:
from scipy.stats import norm

pop_std = 0.2
z_crit = norm.ppf(1 - alpha/2)

margin_error_z = z_crit * (pop_std / np.sqrt(n))
ci_lower_z = mean - margin_error_z
ci_upper_z = mean + margin_error_z

print(f"99% Confidence Interval (z-distribution): ({ci_lower_z:.4f}, {ci_upper_z:.4f})")


99% Confidence Interval (z-distribution): (1.1057, 1.3717)


# **CONCLUSION**

**Both confidence intervals give a range of plausible values for the true mean durability of the print-heads.**

*The t-distribution interval is slightly wider than the z-distribution interval.* This is expected because using the sample standard deviation introduces more uncertainty (especially with a small sample size).

The true mean durability likely lies between:

1.09 to 1.39 million characters (t-dist) or

1.11 to 1.37 million characters (z-dist)

# **Key Takeaway**
When you don’t know the population standard deviation, you use the t-distribution, which gives a wider, more cautious estimate—especially important when working with small samples.

This helps ensure your confidence interval actually contains the true mean more often





